install/Knative-custom-install.md

```diff
@@ -145,197 +143,190 @@ service mesh. If you install any of the following options, you must install the
-| [`in-memory-channel.yaml`][4.3]                      | _Required_        | -                                                          | -               | _Required_         | Only the in-memory channel provisioner.                                             |
-| [`kafka.yaml`][4.4]                                  | _Required_        | -                                                          | -               | _Required_         | Only the Kafka channel provisioner.                                                 |
-| **knative/eventing-sources**                         |                   |                                                            |                 |                    |                                                                                     |
-| [`release.yaml`][5.1]                                | _Required_        | -                                                          | -               | _Required_         | Source types: [Kubernetes][6], [GitHub][6.1], [Container image][6.3]                |
-| [`release-with-gcppubsub.yaml`][5.2]                 | _Required_        | -                                                          | -               | _Required_         | Source types: [Kubernetes][6], [GitHub][6.1], [Container image][6.3], [PubSub][6.4] |
-| [`message-dumper.yaml`][5.3]                         | _Required_        | -                                                          | -               | _Required_         | Event logging service for debugging.                                                |
+**Install files in bold are the recommended standard install suitable for most use cases.**
+
+| Knative Install Filename                         | Notes                                                                                                 |
+| -------------------------------------------------| ----------------------------------------------------------------------------------------------------- |
+| **knative/serving**                              |                                                                                                       |
+| **[`serving.yaml`][1.1]**                        | Installs the  Serving component.                                                                      |
+| **[`monitoring.yaml`][1.2]**                     | Installs the [ELK stack][2], [Prometheus][2.1], [Grafana][2.2], and [Zipkin][2.3]\*                   |
+| [`monitoring-logs-elasticsearch.yaml`][1.3]      | Installs only the [ELK stack][2]\*                                                                    |
+| [`monitoring-metrics-prometheus.yaml`][1.4]      | Installs only [Prometheus][2.1]\*                                                                     |
+| [`monitoring-tracing-zipkin.yaml`][1.5]          | Installs only [Zipkin][2.3]. Requires installation of the ELK stack.\*                                |
```

Comment:
Append filename for clarity:

"...Requires installation of the ELK stack (`monitoring-logs-elasticsearch.yaml`)."